packages/minuteman/extra/dcos-minuteman.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Layer 4 Load Balancer: DC/OS Layer 4 Load Balancing Service
+
+[Service]
+Restart=always
+StartLimitInterval=0
+RestartSec=5
+WorkingDirectory=/opt/mesosphere/active/minuteman/minuteman
+Environment=HOME=/opt/mesosphere
+EnvironmentFile=/opt/mesosphere/etc/minuteman-erl.env
+EnvironmentFile=/opt/mesosphere/etc/check_time.env
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/minuteman.env
+EnvironmentFile=-/run/dcos/etc/minuteman_auth.env
+ExecStartPre=/opt/mesosphere/bin/check-time
+ExecStartPre=/usr/bin/env modprobe ip_vs_wlc
+ExecStartPre=/opt/mesosphere/bin/setup_iptables.sh
+ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-minuteman
+ExecStartPre=/bin/ping -c1 ready.spartan
+ExecStartPre=/bin/ping -c1 leader.mesos
+ExecStartPre=/usr/bin/env mkdir -p /var/lib/dcos/minuteman/mnesia
+ExecStartPre=/usr/bin/env mkdir -p /var/lib/dcos/minuteman/lashup
```

Comment:
@sargun there are multiple systemd instances that are already using `/usr/bin/mkdir` and we'd like to stay consistent / reduce variability. Furthermore, env is not as explicit and introduces potential differences in how the systemd exec might fail.

If you'd like to provide another PR which updates *all* of the hard-coded systemd paths in dcos to use `/usr/bin/env` then we can discuss that in a separate PR but I think it makes sense to follow our recommendation and not introduce a new convention here.